### PR TITLE
[Android] Added a registration view

### PR DESCRIPTION
**Pull request by taabishkathawala**
Moved login to the main screen. Immediately loads Login activity.

Added a registration activity and view that will be triggered when you try to create an account from the login page.

### CODE REVIEW

File: Android/app/src/main/java/com/example/brewhaha_android/Controllers/LoginActivity.kt [hunk 1]
```diff
@@ -1,13 +1,34 @@
...
+
     override fun onCreate(savedInstanceState: Bundle?) {
         super.onCreate(savedInstanceState)
         setContentView(R.layout.activity_login)
+
+        _register_button = findViewById<MaterialButton>(R.id.registerButton)
+        _login_button = findViewById<MaterialButton>(R.id.loginButton)
+        _email_text = findViewById<TextInputEditText>(R.id.emailText)
+        _password_text = findViewById<TextInputEditText>(R.id.passwordText)
+//        _login_button.setOnClickListener()
```
Comment (prabogey) [inline Android/app/src/main/java/com/example/brewhaha_android/Controllers/LoginActivity.kt]: remove comment or add TODO

File: Android/app/src/main/java/com/example/brewhaha_android/Controllers/RegisterActivity.kt [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+        super.onCreate(savedInstanceState)
+        setContentView(R.layout.activity_register)
+
+        _register_button = findViewById(R.id.registerButton) as MaterialButton
+        _input_email = findViewById(R.id.email) as TextInputEditText
+        _input_password = findViewById(R.id.password) as TextInputEditText
+        _username = findViewById(R.id.username) as TextInputEditText
+        _first_name = findViewById(R.id.firstName) as TextInputEditText
+        _last_name = findViewById(R.id.lastName) as TextInputEditText
+//        registerButton.setOnClickListener()
```
Comment (prabogey) [inline Android/app/src/main/java/com/example/brewhaha_android/Controllers/RegisterActivity.kt]: Ditto

File: Android/app/src/main/java/com/example/brewhaha_android/Controllers/LoginActivity.kt (hunk 1 above)
Comment (taabishkathawala) [reply Android/app/src/main/java/com/example/brewhaha_android/Controllers/LoginActivity.kt]: done

File: Android/app/src/main/java/com/example/brewhaha_android/Controllers/RegisterActivity.kt (hunk 2 above)
Comment (taabishkathawala) [reply Android/app/src/main/java/com/example/brewhaha_android/Controllers/RegisterActivity.kt]: done

**Review by prabogey — APPROVED** [review]

**Pull request merged**